CHANGELOG.md

```diff
@@ -51,6 +51,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix the dependency error in copy_vectors within vector_index. 
 - Fix Template substitutions
 - Fix remove un_use _import function
+- Fix Drop options which are not in common MongoDB vs. Ibis 
```

Comment:
Please remove this and change the description of the PR that this is some linting and small refactors.

Reply:
ok